bk-nodes/test_bk_nodes.py

```diff
@@ -0,0 +1,66 @@
+import bk_nodes
```

Comment:
can we make this part of GitHub workflow CI?

Reply:
done